### PR TITLE
<VariableInput /> - onBlur, fix setValue issue

### PR DESCRIPTION
**Pull request by tomershk**
### 🔦 Summary

- New API for `onBlur` to allow client to use different logic than `onSubmit`.
- Fix issue when older value was with new line, focus was on the new line and after we call `setValue` new line was removed. (caused the component to crash)

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- 📚 Change is documented
  - [x] API description
- 🔬 Change is tested
  - [x] Component

### CODE REVIEW

File: src/VariableInput/VariableInput.uni.driver.js [hunk 1]
```diff
@@ -30,6 +30,20 @@ export default (base, body) => {
         contentElement.sendKeys(text);
       }
     },
+    focus: async () => {
```
Comment (nitayneeman) [inline src/VariableInput/VariableInput.uni.driver.js]: Do we really need `focus`? 
I wonder what the use-case is for the consumers (maybe it should be private)?

**Comment by nitayneeman** [comment]
Let's wait for the CI

**Comment by nitayneeman** [comment]
@tomershk We also need to adjust the testkit types:
https://github.com/wix/wix-style-react/blob/master/src/VariableInput/VariableInput.uni.driver.d.ts

**Comment by tomershk** [comment]
Will that resolve this issue as well?
https://wix-style-react.now.sh/?activeTab=Testkit&path=%2Fstory%2Fcomponents-api-components--variableinput

**Comment by nitayneeman** [comment]
> Will that resolve this issue as well?
> https://wix-style-react.now.sh/?activeTab=Testkit&path=%2Fstory%2Fcomponents-api-components--variableinput

This seems to be a different issue

**Pull request merged**